lib/data/remote/models/response/attribute_response.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:built_value/built_value.dart';
+import 'package:built_value/serializer.dart';
+import 'package:vost/data/remote/models/_base/parser.dart';
+
+part 'attribute_response.g.dart';
+
+abstract class AttributeResponse implements Built<AttributeResponse, AttributeResponseBuilder>, SerializedModel<AttributeResponse> {
```

Comment:
My suggestion is to follow the same structured used in the VOST API and have the Attribute object to include the following fields:
```
    attributes:
      type: object
      properties:
        locality:
          type: string
        latitude:
          type: number
          format: double
        longitude:
          type: number
          format: double
        started_at:
          type: string
          format: date-time
        ended_at:
          type: string
          format: date-time
        created_at:
          type: string
          format: date-time
        updated_at:
          type: string
          format: date-time
```

Given most are optional, nullable.

Reply:
What about the `code` and the `name`? Both were in the endpoint response but aren't specified here

Reply:
That was my bad, those should be included, I was trying to find a `set` of attributes that include all fields used everywhere.